utils/manage_audio.py

```diff
@@ -0,0 +1,203 @@
+import argparse
+import numpy as np
+import os
+import pyaudio
```

Comment:
Please create `environment.yml` (conda) or `requirements.txt` (pip) to list all dependencies for easy installation. Feel free to do this for the next PR.